src/javascript/binary/websocket_pages/user/account/authenticate.js

```diff
@@ -1,18 +1,27 @@
-const Client       = require('../../../base/client');
-const BinarySocket = require('../../socket');
+const Client              = require('../../../base/client');
+const BinarySocket        = require('../../socket');
+const DocumentUploader    = require('binary-document-uploader');
+const showLoadingImage    = require('../../../base/utility').showLoadingImage;
+const localize            = require('../../../base/localize').localize;
+const displayNotification = require('../../../base/header').displayNotification;
+const Url                 = require('../../../base/url');
 
 const Authenticate = (() => {
     const onLoad = () => {
         BinarySocket.send({ get_account_status: 1 }).then((response) => {
             if (response.error) {
                 $('#error_message').setVisibility(1).text(response.error.message);
             } else {
-                const get_account_status  = response.get_account_status;
+                const get_account_status = response.get_account_status;
                 const should_authenticate = +get_account_status.prompt_client_to_authenticate;
                 if (should_authenticate) {
                     const status = get_account_status.status;
                     if (!/authenticated/.test(status)) {
-                        $(`#not_authenticated${Client.isAccountOfType('financial') ? '_financial' : ''}`).setVisibility(1);
+                        $('#not_authenticated').setVisibility(1);
+                        if (Client.isAccountOfType('financial')) {
+                            $('#not_authenticated_financial').setVisibility(1);
+                        }
+                        init();
```

Comment:
can we call init first? i.e on line 20, so it sets everything up before making the section visible